charts/csi-secrets-store-provider-aws/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This should be v2 as helm v2 is already deprecated

Reply:
Done